### PR TITLE
Updated readME file

### PR DESCRIPTION
**Pull request by pranavishere2**

### CODE REVIEW

File: example-streamer-uses/README.md [hunk 1]
```diff
@@ -1,66 +1,28 @@
...
-
-```bash
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path/to/vsomeip/lib> cargo run --example uE_client -- --endpoint tcp/0.0.0.0:7444
+    cargo run --bin ue_client
+    cargo run --bin me_service
 ```
+### How to run publish subscribe examples on same local machine
 
-We omit a detail explanation of the expected terminal output as it's a mirror of the `Mechatronics client to high compute service` heading above.
+```
```
Comment (PLeVasseur) [inline example-streamer-uses/README.md:21]: We should have this section look similarly to the RPC section above

* use ## for subheading for PubSub
* use ### for two subsections describing how to run ue_publisher, me_subscriber and then the reverse
* The first section should explain what the intended output should look like and that detail can be omitted from the following section

**Review by PLeVasseur — CHANGES_REQUESTED** [review]
Hey @pranavishere2 -- please take a look

File: example-streamer-uses/README.md [hunk 2]
```diff
@@ -1,66 +1,28 @@
-# Examples
-
-## Running examples
-
-### Running the `up-linux-streamer`
-
-To run one of the examples below and see client and service communicate, you'll need to run the `up-linux-streamer` to bridge between the transports in a terminal. See `up-linux-streamer` README for details.
+# How to run "up-linux-streamer"
+### Run following command from root directory of project:
+     cargo run --bin up-linux-streamer -- --config up-linux-streamer/DEFAULT_CONFIG.json5
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:3]: I tried to use bash code block but it seems that single line does not need bash code block. If we mention ``` then it seems to be visible in the output for single line.

File: example-streamer-uses/README.md [hunk 3]
```diff
@@ -1,66 +1,28 @@
-# Examples
-
-## Running examples
-
-### Running the `up-linux-streamer`
-
-To run one of the examples below and see client and service communicate, you'll need to run the `up-linux-streamer` to bridge between the transports in a terminal. See `up-linux-streamer` README for details.
+# How to run "up-linux-streamer"
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:1]: Done

File: example-streamer-uses/README.md [hunk 4]
```diff
@@ -1,66 +1,28 @@
...
 
-### Mechatronics client to high compute service
+# Examples
+### Please note that these examples should be run in pairs. For instance, ue_service should be paired with me_client, and me_service should be paired with ue_client. Additionally, ensure that up-linux-streamer is running before executing these examples from root directory of project.
 
-Launch the `uE_service` example in another terminal:
+### How to run rpc examples on same local machine. 
 
-```bash
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path/to/vsomeip/lib> cargo run --example uE_service -- --endpoint tcp/0.0.0.0:7445
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:14]: I was not sure if the new implementation is aligned with the old instructions. I have restored this now.

File: example-streamer-uses/README.md [hunk 5]
```diff
@@ -1,66 +1,28 @@
...
 ```
-
-In this example, the "--endpoint" flag will set the endpoint address upon which the zenoh client will listen.
-
-Launch the `mE_client` example in another terminal:
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:19]: Restored

File: example-streamer-uses/README.md [hunk 6]
```diff
@@ -1,66 +1,28 @@
...
-
-```bash
-LD_LIBRARY_PATH=$LD_LIBRARY_PATH:<path/to/vsomeip/lib> cargo run --example mE_client 
+    cargo run --bin ue_service
+    cargo run --bin me_client
 ```
 
-The service and client will run forever. Every second a new message is sent from the mE_client via vsomeip. That vsomeip message is caught and routed over Zenoh to the uE_service. The response makes the same journey in reverse.
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:25]: Restored

File: example-streamer-uses/README.md [hunk 7]
```diff
@@ -1,45 +1,80 @@
+# How to run "up-linux-streamer"
+
+### Run following command from root directory of project:
+```bash
```
Comment (PLeVasseur) [inline example-streamer-uses/README.md:4]: Need newline after all headings

File: example-streamer-uses/README.md [hunk 8]
```diff
@@ -1,45 +1,80 @@
...
+
```
Comment (PLeVasseur) [inline example-streamer-uses/README.md:5]:
```suggestion
```

File: example-streamer-uses/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PLeVasseur) [reply example-streamer-uses/README.md:6]: By which I meant that the ticks ``` should have newlines surrounding them, not inside of them.

No need for extra newlines inside of the code blocks.

File: example-streamer-uses/README.md [hunk 9]
```diff
@@ -1,66 +1,28 @@
...
-### Running the `up-linux-streamer`
-
-To run one of the examples below and see client and service communicate, you'll need to run the `up-linux-streamer` to bridge between the transports in a terminal. See `up-linux-streamer` README for details.
+# How to run "up-linux-streamer"
+### Run following command from root directory of project:
+     cargo run --bin up-linux-streamer -- --config up-linux-streamer/DEFAULT_CONFIG.json5  
 
-### Mechatronics client to high compute service
+# Examples
+### Please note that these examples should be run in pairs. For instance, ue_service should be paired with me_client, and me_service should be paired with ue_client. Additionally, ensure that up-linux-streamer is running before executing these examples from root directory of project.
```
[the remark this answers is not in the document]
Comment (PLeVasseur) [reply example-streamer-uses/README.md:12]: This is not resolved

File: example-streamer-uses/README.md [hunk 10]
```diff
@@ -1,45 +1,80 @@
...
+     cargo run --bin up-linux-streamer -- --config up-linux-streamer/DEFAULT_CONFIG.json5  
+
+```
+
 # Examples
 
-## Running examples
+### Please note that these examples should be run in pairs. For instance, ue_service should be paired with me_client, and me_service should be paired with ue_client. Additionally, ensure that up-linux-streamer is running before executing these examples from root directory of project.
+
+## How to run rpc examples on same local machine.
```
Comment (PLeVasseur) [inline example-streamer-uses/README.md:14]: Why have we now duplicated these sections into this one, without the details about "run this in another terminal", endpoint flag and _another_ one down below that does include that detail, and expected output?

Please revert this change and stick to a setup like this:

# Examples

## RPC

### Mechatronics client to high compute service

Show all the existing content, with any typo issues or ways to run fixed.

### High compute client to mechatronics service

Show all the existing content, with any typo issues or ways to run fixed.

## PubSub

### Mechatronics publisher to high compute subscriber

Mirror "Mechatronics client to high compute service" above, i.e. show how to run this as well as sample output to the user can see what's expected output

### High compute publisher to mechatronics subscriber

Show how to run this, feel free to put a disclaimer as I had in which we say we omit the details.

**Review by PLeVasseur — CHANGES_REQUESTED** [review]
Hey @pranavishere2 -- take a look

File: example-streamer-uses/README.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:12]: Done

File: example-streamer-uses/README.md [hunk 11]
```diff
@@ -1,66 +1,28 @@
...
-
-### Running the `up-linux-streamer`
-
-To run one of the examples below and see client and service communicate, you'll need to run the `up-linux-streamer` to bridge between the transports in a terminal. See `up-linux-streamer` README for details.
+# How to run "up-linux-streamer"
+### Run following command from root directory of project:
+     cargo run --bin up-linux-streamer -- --config up-linux-streamer/DEFAULT_CONFIG.json5  
 
-### Mechatronics client to high compute service
+# Examples
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:10]: Done

File: example-streamer-uses/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md:6]: Done

File: example-streamer-uses/README.md [hunk 12]
```diff
@@ -1,66 +1,28 @@
...
+### Please note that these examples should be run in pairs. For instance, ue_service should be paired with me_client, and me_service should be paired with ue_client. Additionally, ensure that up-linux-streamer is running before executing these examples from root directory of project.
 
-Launch the `uE_service` example in another terminal:
+### How to run rpc examples on same local machine.
```
[the remark this answers is not in the document]
Comment (pranavishere2) [reply example-streamer-uses/README.md]: Done

File: example-streamer-uses/README.md (hunk 7 above)
Comment (pranavishere2) [reply example-streamer-uses/README.md:4]: Done

File: example-streamer-uses/README.md (hunk 8 above)
Comment (pranavishere2) [reply example-streamer-uses/README.md:5]: I am assuming this suggested change is to remove extra newline from code block. It is done.

**Comment by PLeVasseur** [comment]
Appreciate the help with this @pranavishere2. I think this is now fixed by #58, so closing.

**Pull request closed without merging**